client/pages/paperwork/[rescueId]/index.js

```diff
@@ -303,10 +303,14 @@ class Paperwork extends React.Component {
           <span className="date-created content">{formatAsEliteDateTime(rescue.attributes.createdAt)}</span>
           <span className="label">Updated</span>
           <span className="date-updated content">{formatAsEliteDateTime(rescue.attributes.updatedAt)}</span>
-          <span className="label">IRC Nick</span>
-          <span className="irc-nick content">{rescue.attributes.data.IRCNick}</span>
-          <span className="label">Language</span>
-          <span className="language content">{rescue.attributes.data.langID}</span>
+          {(rescue.attributes.data !== null) && (
```

Comment:
No need for explicit null checks.
```suggestion
          {Boolean(rescue.attributes.data) && (
```